wdn/templates_5.1/js-src/slideshows.babel.js

```diff
@@ -0,0 +1,124 @@
+require(['dcf-utility', 'dcf-slideshow', 'css!js-css/slideshows'], () => {
+  const openCaptionEvent = new Event('openCaption');
+  const closeCaptionEvent = new Event('closeCaption');
+  const slideshows = document.querySelectorAll('.dcf-slideshow');
+  const uls = document.querySelectorAll('.dcf-slideshow ul');
+
+  const slideshow = new DCFSlideshow(slideshows, uls, openCaptionEvent, closeCaptionEvent);
+
+  slideshow.initialize();
+
+  const keyframesClose1 = [
+    {
+      transform: 'rotate(45deg)',
+      transformOrigin: '50% 50%'
+    },
+    {
+      transform: 'rotate(0deg)',
+      transformOrigin: '50% 50%'
+    }
+  ];
+  const keyframesClose2 = [
+    {
+      transform: 'rotate(-45deg)',
+      transformOrigin: '50% 50%'
+    },
+    {
+      transform: 'rotate(0deg)',
+      transformOrigin: '50% 50%'
+    }
+  ];
+  const keyframesOpen1 = [
+    {
+      transform: 'rotate(0deg)',
+      transformOrigin: '50% 50%'
+    },
+    {
+      transform: 'rotate(45deg)',
+      transformOrigin: '50% 50%'
+    }
+  ];
+  const keyframesOpen2 = [
+    {
+      transform: 'rotate(0deg)',
+      transformOrigin: '50% 50%'
+    },
+    {
+      transform: 'rotate(-45deg)',
+      transformOrigin: '50% 50%'
+    }
+  ];
+  const options = {
+    duration: 250,
+    fill: 'forwards'
+  };
+
+  Array.prototype.forEach.call(slideshows, (slideshow) => {
+    // Slideshow Controls
+    let ctrlGroup = slideshow.querySelector('.slide-show-control'); // DCF
+    let ctrls = ctrlGroup.querySelectorAll('li');
+
+    Array.prototype.forEach.call(ctrls, (ctrl) => {
+      if (ctrl.classList.contains('slide-prev-btn')) {
+        let ctrlButton = ctrl.querySelector('button');
+        ctrlButton.classList.add('dcf-d-flex', 'dcf-ai-center', 'dcf-pt-4', 'dcf-pb-4', 'dcf-inverse');
+        ctrlButton.innerHTML =
+          '<svg class="dcf-h-4 dcf-w-4 dcf-fill-current" width="24" height="24" viewBox="0 0 24 24" aria-hidden="true">' +
+          '<path d="M21.746.064a.504.504 0 0 0-.504.008l-19 11.5a.499.499 0 0 0-.001.856l19' +
+          ' 11.5A.501.501 0 0 0 22 23.5V.5a.5.5 0 0 0-.254-.436z"></path>' +
```

Comment:
I've updated the previous arrow icon: `'<path d="M23.509 9.856c-.38-.55-.928-.852-1.542-.852H9.74l4.311-4.151c.995-.994.961-2.646-.074-3.682-1.001-1-2.722-1.033-3.68-.077L.148 11.144a.5.5 0 00-.003.707l9.978 10.079a2.445 2.445 0 001.737.705c.707 0 1.407-.294 1.92-.806a2.737 2.737 0 00.807-1.923 2.431 2.431 0 00-.708-1.733l-4.156-4.16h12.276c.618 0 1.161-.302 1.53-.851.304-.451.471-1.041.471-1.658 0-.596-.179-1.196-.491-1.648z"></path>' +`